app/code/Magento/Catalog/Test/Unit/Block/Product/ProductList/ToolbarTest.php

```diff
@@ -7,19 +7,21 @@
 
 namespace Magento\Catalog\Test\Unit\Block\Product\ProductList;
 
+use PHPUnit\Framework\TestCase;
```

Comment:
Seems like earlier all imports were sorter, while now they're not. Could you run optimize imports in phpstorm for all changes files and commit?